www/base/src/app/common/directives/buildsummary/buildsummary.tpl.jade

```diff
@@ -1,32 +1,34 @@
 .panel.panel-default(ng-class="results2class(buildsummary.build)")
   .panel-heading.no-select
-    .btn.btn-xs.btn-default(ng-click="buildsummary.toggleFullDisplay()", title="Expand all step logs")
-        i.fa.fa-chevron-circle-right.rotate(ng-class="{'fa-rotate-90':buildsummary.fulldisplay}")
-    .btn.btn-xs.btn-default(ng-click="buildsummary.toggleDetails()", title="Show steps according to their importance")
-        i.fa.fa-expand
-        {{buildsummary.levelOfDetails()}}
-    | &nbsp;
-    span(ng-if="buildsummary.prefix") {{buildsummary.prefix}}&nbsp;
-    a(ui-sref="build({builder:buildsummary.builder.builderid, build:buildsummary.build.number})")
-      | {{buildsummary.builder.name}}/{{buildsummary.build.number}}
-    .pull-right
-      span(ng-show="buildsummary.build.complete")
-          | {{(buildsummary.build.complete_at - buildsummary.build.started_at)| durationformat:'LLL' }}
-      span(ng-show="!buildsummary.build.complete")
-          | {{(buildsummary.now - buildsummary.build.started_at)| durationformat:'LLL' }}
-      span
-        | &nbsp;{{buildsummary.build.state_string}}&nbsp;
-      .label(ng-class="results2class(buildsummary.build)")
-        | {{results2text(buildsummary.build)}}
-      span(ng-if="buildsummary.parentbuild")
-          | &nbsp;
-          a.label(ng-class="results2class(buildsummary.parentbuild)",
-          ui-sref="build({builder:buildsummary.parentbuilder.builderid, build:buildsummary.parentbuild.number})")
-                | {{buildsummary.parentrelationship}}:
-                | {{buildsummary.parentbuilder.name}}/{{buildsummary.parentbuild.number}}
+    .row
+      .col-sm-8
```

Comment:
By removing row and col text are not aligned anymore:
<img width="699" alt="without_row_and_col" src="https://cloud.githubusercontent.com/assets/12556701/14597870/a481d248-054f-11e6-9974-79144b11b618.png">

I propose to split the columns with same size (col-sm-6) as folllow:
<img width="681" alt="with_row_and_col" src="https://cloud.githubusercontent.com/assets/12556701/14597888/c06f2f14-054f-11e6-9ea9-c031d9c3a185.png">